rllib/policy/tf_policy.py

```diff
@@ -493,9 +494,34 @@ def num_state_tensors(self) -> int:
 
     @override(Policy)
     @DeveloperAPI
-    def get_state(self) -> Union[Dict[str, TensorType], List[TensorType]]:
+    def get_state(self) -> PolicyState:
         # For tf Policies, return Policy weights and optimizer var values.
         state = super().get_state()
+
+        # Add this Policy's spec so it can be retreived w/o access to the original
```

Comment:
Add PolicySpec AND - if possible - native keras/torch saved model files to returned state.